matrix_client/api.py

```diff
@@ -179,6 +188,10 @@ def create_room(self, alias=None, is_public=False, invitees=None):
             content["room_alias_name"] = alias
         if invitees:
             content["invite"] = invitees
+        if name:
+            content["name"] = name
+        if federate is not None:
```

Comment:
wrt above comment: ``if federate is False``

Reply:
feredate can be True, False or None. If it's None we do not add it to json.